dstep/driver/Application.d

```diff
@@ -34,30 +38,40 @@ class Application : DStack.Application
 
     private
     {
-        string[] inputFiles;
-
-        Index index;
-        TranslationUnit translationUnit;
-        DiagnosticVisitor diagnostics;
-
+        bool helpFlag;
         Language language;
         string[] argsToRestore;
-        bool helpFlag;
         Compiler compiler;
     }
 
     protected override void run ()
     {
         handleArguments;
-
         if (arguments.argument.input.hasValue)
         {
-            inputFiles ~= arguments.argument.input;
-            startConversion(inputFiles.first);
+            int j = 0;
+            auto remArgs = remainingArgs;
```

Comment:
You're not really supposed to use `remainingArgs` like this. Instead you can access all files passed on the command line as `arguments.argument.input.values`. I think you need to increase the maximum numbers of allowed values for the `input` argument in the `setupArguments` method. See [this](https://github.com/jacob-carlborg/mambo/blob/master/mambo/arguments/Arguments.d#L264) for reference.